mora/integrations/serviceplatformen.py

```diff
@@ -7,6 +7,7 @@
 #
 
 import random
+import service_person_stamdata_udvidet
```

Comment:
Jeg er ikke den store fan af det pakkenavn — er det noget der også bruges andetsteds, og dermed hugget i sten?

Reply:
Det er ikke noget jeg har valgt. Det var så vidt jeg husker pakkens originale navn da den lå i AAK-repo'et, så det arvede den vist da den blev løftet ud som et separat repo.

Reply:
Men det bliver ikke brugt andre steder endnu, af hvad jeg ved. Så lad os ændre det. Forslag?

Reply:
1) Vi vil gerne have dette op på PyPI, så det skal være et rimeligt unikt og fast navn.
2) Det må gerne sige eller indikere noget med ServicePlatformen, uden nødvendigvis at bruger det navn. (Må vi det?)
3) Det behøver slet ikke at være så specifikt — på sigt kan vi ligeså godt klemme alt SP-relevant i den samme pakke.
4) [pysp](https://pypi.org/project/pysp/) er ledigt, så vidt jeg kan se.

Reply:
Integrationerne til de enkelte services på ServicePlatformen er løftet ud som separate moduler.
Idéen bag navnet, er at det svarer til det navn den specifikke service har.

Reply:
Er der nogen grund til at have dem i separate pakker på PyPI?

Reply:
@agger-magenta Har du nogen holdning til det her?

Reply:
Tilsvarende har vi [service_cvr_online](https://github.com/magenta-aps/service_cvr_online)

Reply:
Personligt vil jeg mene at du burde være i ét — de laver stort set det samme. Men det er ikke så vigtigt for denne PR; flet den bare!